daemon/algod/api/algod.oas3.yml

```diff
@@ -2006,7 +2006,7 @@
     },
     "/v2/accounts/{address}/assets/{asset-id}": {
       "get": {
-        "description": "Given a specific account public key, this call returns the account's asset holding and asset params (if they exist) about a given asset ID",
+        "description": "Given a specific account public key and asset ID, this call returns the account's asset holding and asset parameters (if either exist). Asset parameters will only be returned if the provided address is the asset's creator.",
```

Comment:
I'd suggest adding the info about the conditions necessary for asset holding vs asset param fields to be present directly to the field descriptions in `AccountAssetResponse` and `AccountApplicationResponse`

Reply:
OK updated those desriptions